osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/entities/DeviceKeyProcessing.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.entities;
+
+import java.time.Instant;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class DeviceKeyProcessing {
+
+  @Id
+  @Column(length = 40)
+  private String deviceIdentification;
+
+  @Column private Instant startTime;
+
+  public String getDeviceIdentification() {
+    return this.deviceIdentification;
+  }
+
+  public void setDeviceIdentification(final String deviceIdentification) {
+    this.deviceIdentification = deviceIdentification;
+  }
+
+  public java.time.Instant getStartTime() {
```

Comment:
Cleaner style would be not to use the fully qualified name `java.time.Instant` here, but just use `Instant` like elsewhere.

Reply:
Removed package name